upstream-community-operators/oneagent/dynatrace.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: oneagent
```

Comment:
Has oneagent been tested against kubernetes?

Reply:
It has with mixed results, but I haven't heard if it was reproducible or not.